tests/test_levelset.py

```diff
@@ -0,0 +1,93 @@
+import unittest
+import firedrake as fd
+import fireshape as fs
+import fireshape.zoo as fsz
+import ROL
+
+class LevelsetTest(unittest.TestCase):
+
+    def run_levelset_optimization(self, Q, write_output=False):
+        """ Test template for fsz.LevelsetFunctional."""
+
+        #tool for developing new tests, allows storing shape iterates
+        if write_output:
+            out = fd.File("T.pvd")
```

Comment:
not sure this is what we want to print to check whether a test does what is should